src/main/java/org/jenkinsci/plugins/plaincredentials/impl/FileCredentialsImpl.java

```diff
@@ -159,7 +150,7 @@ public FileCredentialsImpl(@CheckForNull CredentialsScope scope, @CheckForNull S
         }
         if (LOGGER.isLoggable(Level.FINE)) {
             LOGGER.log(Level.FINE, "for {0} have {1} of length {2} after upload of ‘{3}’",
-                    new Object[]{getId(), this.fileName, this.secretBytes.getPlainData().length, name});
+                    new Object[]{getId(), this.fileName, this.secretBytes.getPlainData().length, file.getName()});
```

Comment:
Will this not throw NPE when `file == null`, as according to the constructor annotation it is permitted to?